bitnami/contour/templates/rbac.yaml

```diff
@@ -3,117 +3,94 @@
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
-  name: {{ include "contour.fullname" .}}:contour
-  labels: {{- include "contour.labels" . | nindent 4 }}
+  name: {{ include "common.names.fullname" .}}:contour
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
 rules:
 - apiGroups:
   - ""
   resources:
   - configmaps
-  - endpoints
-  - nodes
-  - pods
-  - secrets
-  verbs:
-  - list
-  - watch
-- apiGroups:
-  - ""
-  resources:
-  - nodes
   verbs:
+  - create
   - get
+  - update
 - apiGroups:
   - ""
   resources:
-  - services
+  - endpoints
   verbs:
   - get
   - list
   - watch
 - apiGroups:
-  - "networking.k8s.io"
+  - ""
   resources:
-  - ingresses
+  - secrets
   verbs:
   - get
   - list
   - watch
 - apiGroups:
-  - "networking.k8s.io"
+  - ""
   resources:
-  - "ingresses/status"
+  - services
   verbs:
   - get
   - list
   - watch
-  - patch
-  - post
-  - update
-- apiGroups: ["contour.heptio.com"]
-  resources: ["ingressroutes", "tlscertificatedelegations"]
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - gatewayclasses
+  - gateways
+  - httproutes
+  - tcproutes
   verbs:
   - get
   - list
   - watch
-  - put
-  - post
-  - patch
-- apiGroups: ["projectcontour.io"]
-  resources: ["httpproxies", "tlscertificatedelegations"]
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingresses
   verbs:
   - get
   - list
   - watch
-  - put
-  - post
-  - patch
 - apiGroups:
-  - "projectcontour.io"
+  - networking.k8s.io
   resources:
-  - "httpproxies/status"
-  verbs:
-  - update
-- apiGroups: ["networking.x.k8s.io"]
-  resources: ["gatewayclasses", "gateways", "httproutes", "tcproutes"]
+  - ingresses/status
   verbs:
+  - create
   - get
-  - list
-  - watch
-  - put
-  - post
-  - patch
----
-apiVersion: rbac.authorization.k8s.io/v1beta1
-kind: Role
-metadata:
-  name: {{ include "contour.fullname" .}}:contour-leaderelection
-  labels: {{- include "contour.labels" . | nindent 4 }}
-rules:
+  - update
 - apiGroups:
-  - ""
+  - projectcontour.io
   resources:
-  - configmaps
+  - httpproxies
+  - tlscertificatedelegations
   verbs:
-  - create
   - get
   - list
   - watch
-  - update
 - apiGroups:
-  - ""
+  - projectcontour.io
   resources:
-  - events
+  - httpproxies/status
   verbs:
   - create
+  - get
   - update
-  - patch
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: Role
 metadata:
-  name: {{ include "contour.fullname" .}}:contour-certgen
-  labels: {{- include "contour.labels" . | nindent 4 }}
+  name: {{ include "common.names.fullname" .}}:contour-certgen
```

Comment:
do you know if there is any reason to use `:` instead of `-` as in the rest of the templates? please change it for the rest if there is no strong reason in order to be aligned.

Reply:
Uff I simply do not know :) Maybe we need to ask the initial chart contributor (@Cellebyte ?).

Reply:
No it was adapted from an old chart you could change it :)

Reply:
Okay :+1: Already updated it.